src/openscm_runner/adapters/ciceroscm_adapter/make_scenario_files.py

```diff
@@ -0,0 +1,236 @@
+"""
+Module with functionality to make emission input files
+"""
+import csv
+import logging
+import os
+
+import numpy as np
+import pandas as pd
+
+# Method to convert the unit names form tonnes to grams
```

Comment:
This comment should be turned into the docstring for `unit_name_converter`

Reply:
It more or less already was. I've removed the comment